bin/kcluster_parallel.R

```diff
@@ -0,0 +1,141 @@
+# modification of kCLuster (clusterSeq package) to include 'ncores' parameter in bplapply:
```

Comment:
@pmb59 please open an issue on clusterSeq to see if they can incorporate this.

Reply:
For reference: https://github.com/tjh48/clusterSeq/issues/1